backend/simulation/draw/visualiser.py

```diff
@@ -1,3 +1,5 @@
+# mypy: ignore-errors
```

Comment:
?
seems to work fine without this (just tested)

Reply:
Strange, broke locally. Might be something wrong with my installation of pygame (error surface.Surface doesn't match Surface).